geoalchemy2/types.py

```diff
@@ -84,28 +86,43 @@ class _GISType(UserDefinedType):
     """ The name of ST_*FromText function for this type.
         Set in subclasses. """
 
+    as_binary = None
+    """ The name of ST_*AsBinary function for this type.
+        Set using the option ``use_ewkb`` or in subclasses. """
+
     comparator_factory = Comparator
     """ This is the way by which spatial operators are defined for
         geometry/geography columns. """
 
     def __init__(self, geometry_type='GEOMETRY', srid=-1, dimension=2,
-                 spatial_index=True, management=False):
+                 spatial_index=True, management=False, use_ewkb=False):
```

Comment:
Thinking about it, the `use_ewkb` constructor argument should not be on `_GISType`, it should be on `Geometry` instead.